sickbeard/helpers.py

```diff
@@ -1561,3 +1561,14 @@ def canonical_name(obj, fmt='{key}:{value}', separator='|', ignore_list=frozense
     """
     guess = obj if isinstance(obj, dict) else guessit.guessit(obj)
     return str(separator.join([fmt.format(key=k, value=v) for k, v in guess.items() if k not in ignore_list]))
+
+
+def get_broken_providers():
+    """Get broken providers from cdn.pymedusa.com."""
+    url = 'https://cdn.pymedusa.com/providers/broken_providers.json'
+    response = getURL(url, session=make_session(), returns='json')
+    if not response:
+        logger.debug('Unable to update broken providers')
+        return []
+
+    return list(response)
```

Comment:
@ratoaq2 you are right. sorry: `DEBUG::EVENT-QUEUE :: [f3ebd83] response type: <type 'list'>
`